osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -73,14 +102,31 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             SelectedItem.BindValueChanged(_ => Scheduler.AddOnce(selectedItemChanged));
-            SelectedItem.Value = Playlist.FirstOrDefault();
+
+            Playlist.BindCollectionChanged(onPlaylistChanged, true);
```

Comment:
Keep in mind this is not correct anymore and will break multiplayer. Tempted to say this shouldn't exist at all.

Reply:
Yep, should be gone.